calico_containers/adapter/netns.py

```diff
@@ -192,7 +192,7 @@ def set_up_endpoint(ip, cpid, next_hop_ips,
 
     # Return an Endpoint.
     network = IPNetwork(IPAddress(ip))
-    ep = Endpoint(ep_id=ep_id, state="active", mac=mac, if_name=veth_name)
+    ep = Endpoint(hostname, "docker", cpid, endpoint_id, "active", mac)
```

Comment:
This needs work. Now that endpoint objects need hostname, orchestrator_id, and state, this function needs to pass them in. Is this script *always* run by docker, so the orchestrator ID is set here? Since its creating a new node, is its state active? Or does it need to be provided?